app/settings.py

```diff
@@ -0,0 +1,6 @@
+DAY = "%Y-%m-%d"
+
+FORMATS = [DAY, "%Y-%m-%d/%H:%M:%S", "%Y-%m-%d/%H:%M"]
```

Comment:
What is the intention of multiple formats? Why not restrict user to always use the same format for all requests?

Reply:
2019-10-18, 2019-10-19,
2019-10-1/21:23 2019-10-1/21:25
Its done for user's convenience.